assets/scss/layout-builder.scss

```diff
@@ -557,6 +557,9 @@ figure.drupal-media.ck-widget {
   }
 }
 .ui-dialog {
+  &.entity-browser-modal {
+    top: 8% !important;
```

Comment:
This breaks the ability for a user to reposition the modal on the page in an unexpected way. 

https://github.com/YCloudYUSA/y_lb/assets/238201/2cd28e21-4161-4425-b8d2-11420d285ef4

I'd recommend reviewing https://www.drupal.org/project/issues/entity_browser?text=modal&status=All and finding a more complete solution.
https://github.com/YCloudYUSA/y_lb/assets/238201/2cd28e21-4161-4425-b8d2-11420d285ef4

Reply:
We need less !important statements out there